mantidproject/mantid
# Migrate ColorBarWidget from MantidQt/SliceViewer to MantidQt/MantidWidgets

Fixes #15276 

No release notes

Testing:
- Do a <b>clean</b> build.
- open the SliceViewer, load your favorite workspace and verify the functionality of the  color bar with the following actions:
1. change the scaling (linear, logarithm, power)
2. change the minimum and maximum range.
3. change the color map (double click on the color bar to open the file dialog that enables you to load a different color map)
- Verify that the widget is available in Qt-Designer:
1. build target <code>DesignerPlugins</code> (<code>make -j666 DesignerPlugins</code>).
2. In a terminal, set environment variable <code>QT_PLUGIN_PATH=your_build_directory/bin</code>.
3. In the same terminal, start Qt-designer (<code>$> designer-qt4</code>)
4. In the Qt-Designer menu, go to <i>Help -> About Plugins</i>, then verify the ColorBarWidget is listed (probably at the end of the list)
5. In the <b>Widget Box</b>, find the ColorBarWidget (again, probably at the end)
6. Create a new Widget and add the ColorBarWidget onto it.
